datafusion/core/src/catalog/mod.rs

```diff
@@ -20,7 +20,7 @@
 
 #![allow(clippy::module_inception)]
 pub mod catalog;
-pub mod information_schema;
+pub(crate) mod information_schema;
```

Comment:
There wasn't anything public in the module so we might as well just make it crate private

https://docs.rs/datafusion/latest/datafusion/catalog/information_schema/index.html